javascript/javascript1/week2/weather_Wear.js

```diff
@@ -0,0 +1,27 @@
+
+function Temperature (tempToDay)
+{
+    if((tempToDay < -30) || (tempToDay > 51))
+    {
+        return "It is to hot/cold, we can not life in this temperature !!";
+    }
+    else if((tempToDay < -20) || (tempToDay < 0))
```

Comment:
this line translates to `if temperature is less than -20 **or** temperature is less then 0`. i think you want to use something like this ? an expression with **and** operator
```suggestion
    else if((tempToDay > -20) && (tempToDay < 0))
```